README.md

```diff
@@ -120,155 +128,221 @@ Client(address='', container='', user='', password='', token='')
-    See [User Authentication](#user-authentication).
 
 <a id="client-constructor-return-value"></a>
 #### Return Value
 
 Returns a new Frames `Client` data object.
 
-<a id="client-constructor-example"></a>
-#### Example
+<a id="client-constructor-examples"></a>
+#### Examples
 
-The following example, for local platform execution, creates a Frames client for accessing data in the "users" container by using the authentication credentials of the user "iguazio":
+The following examples, for local platform execution, both create a Frames client for accessing data in the "users" container by using the authentication credentials of user "iguazio"; the first example uses access-key authentication while the second example uses username and password authentication (see [User Authentication](#user-authentication)):
+
+```python
+import v3io_frames as v3f
+client = v3f.Client("framesd:8081", user="iguazio", token="e8bd4ca2-537b-4175-bf01-8c74963e90bf", container="users")
+```
 
 ```python
 import v3io_frames as v3f
 client = v3f.Client("framesd:8081", user="iguazio", password="mypass", container="users")
 ```
 
 <a id="client-common-method-params"></a>
-### Common Client Method Parameters
+### Common `Client` Method Parameters
 
-All client methods receive the following common parameters:
+All client methods receive the following common parameters; additional, method-specific parameters are described for each method.
 
 - <a id="client-method-param-backend"></a>**backend** &mdash; The backend data type for the operation.
-  See the backend-types descriptions in the [overview](#backend-types).
+  See [Backend Types](#backend-types).
 
   - **Type:** `str`
-  - **Valid Values:** `"csv"` | `"kv"` | `"stream"` | `"tsdb"`
   - **Requirement:** Required
+  - **Valid Values:** `"csv"` (for testing) | `"kv"` | `"stream"` | `"tsdb"`
 
-- <a id="client-method-param-table"></a>**table** &mdash; The relative path to the backend data &mdash; A directory in the target platform data container (as configured for the client object) that represents a TSDB or NoSQL table or a data stream.
+- <a id="client-method-param-table"></a>**table** &mdash; The relative path to the backend data &mdash; a directory in the target platform data container (as configured for the client object) that represents a TSDB or NoSQL table or a data stream.
   For example, `"mytable"` or `"examples/tsdb/my_metrics"`.
 
   - **Type:** `str`
   - **Requirement:** Required unless otherwise specified in the method-specific documentation
 
-Additional method-specific parameters are described for each method.
-
 <a id="method-create"></a>
-### create Method
+### `create` Method
 
-Creates a new TSDB table or a stream in a platform data container, according to the specified backend type.
+Creates a new TSDB table or stream in a platform data container, according to the specified backend type.
```

Comment:
Capital `Stream`

Reply:
?! There's no reason to capitalize "stream" here; we don't capitalize it anywhere else (in the Frames or platforms doc).